ckanext/canada/tables/briefingt.yaml

```diff
@@ -303,6 +303,142 @@ resources:
       tracking_number: 1089-4266
     sort: tracking_number asc
 
+- title: Proactive Publication - Briefing Note Titles and Numbers Nothing to Report
+  resource_name: briefingt-nil
+  create_form: true
+  edit_form: false #edit not possible for nil reports since only stored data is primary key
+  fields:
+
+  - datastore_id: year
+    datastore_type: year
+    label: Year
```

Comment:
Should this say "Fiscal Year" and represented ? Does "2022 P10" mean 2023-01 or 2022-01? Seems likely to be confusing

Reply:
Good point @wardi. For this template @PatLittle has confirmed it is calendar year.